benchmarks-api/src/main/java/io/scalecube/benchmarks/BenchmarkState.java

```diff
@@ -148,6 +148,7 @@ public final void shutdown() {
     if (!started.compareAndSet(true, false)) {
       throw new IllegalStateException("BenchmarkState is not started");
     }
+    LOGGER.info("Benchmarks is shutdown...");
```

Comment:
`Benchmark is shutting down ...`  
few lines below must be 
`Benchmark has shut down`

Reply:
done